code/modules/antagonists/ninja/weapons/energy_glaive.dm

```diff
@@ -0,0 +1,110 @@
+/**
+ * # Energy Glaive
+ *
+ * The space ninja's glaive.
+ *
+ * The glaive that only space ninja spawns with.  Comes with 12 force and 40 throwforce, along with a throw-teleport system.
+ * Throwing glaive will teleport thrower to where it arrived. Knockdown people that stay on turf where user teleport.
+ * 4 charges witch 10 SECONDS cooldown.
+ * Make two attack instead of one. Target will be chosen from any living mob in 3x3 aoe of user.
+ * If no one in range glaive will make second attack to first target again.
+ *
+ */
+#define GLAIVE_COOLDOWN 10 SECONDS
+/obj/item/energy_glaive
+	name = "energy glaive"
+	desc = "Double-sided energy spear."
```

Comment:
Make the description a little bit cooler. Something like "This glaive is used among guards of the Spider Clan to decimate attacking clans with it's double-bladed design. A small microchip the size of a nail is implanted within the wrist of users of this glaive, which allows the user to invert itself through bluespace to the position of holding the spear. The microchip is activated upon the spear impacting with an object after being thrown.